platform/platform-api/src/main/java/org/codice/ddf/platform/services/common/Describable.java

```diff
@@ -26,7 +28,7 @@
      *
      * @return the version of the item being described (example: 1.0)
      */
-    String getVersion();
+    @NotNull String getVersion();
```

Comment:
As much as I love a clear API, it is a bit redundant. One of our guiding principles is to never return `null` whenever possible. No point in adding verbosity to communicate a standard practice. Better to use the `@Nullable` annotation only for exceptions to the rule. I would remove the `@NotNull` annotations.